src/MongoDbEventStoreAdapter.php

```diff
@@ -72,6 +106,8 @@ public function __construct(\MongoClient $mongoClient, $dbName, $streamCollectio
         if ($streamCollectionName) {
             $this->streamCollectionName = $streamCollectionName;
         }
+
+        $this->writeConcern = array_merge($this->writeConcern, $writeConcern);
```

Comment:
hhm, can this cause trouble if one is not aware that writeConcern is merged instead of replaced? Should we document it?